### DISCUSSION TITLE
스터디 - 1주차 스터디 목록

### DISCUSSION THREAD

**Discussion by vi2920va** [vi2920va/FE_interview#6] [General]
###  일정 
- 2021.11.11(목) 
- 시간 15:00 ~ 16:00
 
### 진행 방식
- 입장 확인
- PR 리뷰
- 모의 면접 진행(진행 후 피드백 필수!)
- 개선사항/ 건의사항 자유롭게 대화
- 회고록 작성

###  스터디 모음
이송현 
 - JS (동기 처리와 비동기 처리)
 - JS (Promise, Callback)
 - React(SPA / CSR)

이동기
- JS (Scope)

조영생
- JS (This)
- Web(브라우저의 작동원리)

**Comment by vi2920va** [comment]
회고록
=== 
- 스터디 목표
- 소감 한 마디
- 건의사항

**Reply by ki7348** [reply]
스터디 목표: 꾸준한 실력 향상
소감 한 마디: 팀원들과 첫 소통을 해서 좋았습니다
건의사항: x

**Reply by vi2920va** [reply]
- 스터디 목표 : 유연하게 의사소통하면서 개발하는 개발자가 되고 싶습니다.
- 소감 한 마디 : 팀원들과 만나서 피드백을 주고 받아서 좋았습니다.
- 건의 사항: x

**Reply by whdudtod1273** [reply]
스터디 목표 : 내가 알고 있는 것 남한테 정확하게 설명할 수 있게 하기
소감 한 마디 : 첫날이었지만 실제 면접 느낌도 났고 좋은 피드백들을 해주셔서 좋았습니다. 
건의 사항 : x
